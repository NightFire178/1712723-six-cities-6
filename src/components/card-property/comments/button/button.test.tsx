import  React from 'react'
import {render} from '@testing-library/react'
import ComponentButton from "./button";

jest.mock(`react-hook-form`)
test(`button card-property comments component block test`, ()=>{
  const mockControl = jest.fn()
  //eslint-disable-next-line
  // @ts-ignore
  const {container} = render(<ComponentButton control={mockControl}/>)
  expect(container).toMatchSnapshot()
})
