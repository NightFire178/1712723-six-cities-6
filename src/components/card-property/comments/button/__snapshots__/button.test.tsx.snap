// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button card-property comments component block test 1`] = `
<div>
  <div
    class="reviews__button-wrapper"
  >
    <p
      class="reviews__help"
    >
      To submit review please make sure to set 
      <span
        class="reviews__star"
      >
        rating
      </span>
       and describe your stay with at least 
      <b
        class="reviews__text-amount"
      >
        50 characters
      </b>
      .
    </p>
    <button
      class="reviews__submit form__submit button"
      disabled=""
      style="pointer-events: none;"
    >
      Submit
    </button>
  </div>
</div>
`;
